website/client/components/settings/site.vue

```diff
@@ -352,6 +349,11 @@ export default {
       alert(this.$t(`${attribute}Success`));
       this.user[attribute] = updates[attribute];
     },
+    async changeDisplayName (newName) {
+      await axios.put('/api/v4/user/', {'profile.name': newName});
+      alert(this.$t('displayNameSuccess'));
+      this.user.profile.name = newName;
+    },
```

Comment:
we should reset `this.displayName` here or it'll be stuck to the older version (before the change)